src/wwwroot/controllers/MainCtrl.js

```diff
@@ -28,8 +28,11 @@
       return 1;
     };
      
+    $rootScope.getFreeTrialNotificationFromStorage = auth.getFreeTrialNotificationFromStorage();
```

Comment:
Maybe `getFreeTrialNotificationFromStorage` the right name for the variable could be `freeTrialNotificationDate`.

Also, maybe you could store it in a local variable in place of `rootScope`